clients/gtest/gemm_gtest.cpp

```diff
@@ -44,6 +44,8 @@ Representative sampling is sufficient, endless brute-force sampling is not neces
 const
 vector<vector<int>> matrix_size_range = {
                                         {-1, -1, -1, -1, 1, 1},
+                                        { 3, 33,  3,  3, 3, 3},
```

Comment:
{600, 500, 500, xxx} are is valid for certain transpose but invalid for other transpose but should pass as the argument checker will filter it out.  We purposely make some artificial ones.